compiler/testData/codegen/bytecodeText/deadCodeElimination/simpleConstructorNotRedundant.kt

```diff
@@ -1,8 +1,8 @@
 class A
-fun box() {
+fun box(u: Int) {
```

Comment:
and this?

Reply:
Similarly, the check and throw for y!! would be optimized away without this.